test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using AndcultureCode.CSharp.Testing.Extensions;
+using AndcultureCode.CSharp.Testing.Tests;
+using AndcultureCode.CSharp.Web.Extensions;
+using Microsoft.AspNetCore.Diagnostics;
+using Moq;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AndcultureCode.CSharp.Core.Tests.Unit.Extensions
```

Comment:
I believe this namespace is incorrect; shouldn't it be `AndcultureCode.CSharp.Web.Tests.Extensions`?

Reply:
snap, good catch